typescript/vrotsc/src/compiler/program.ts

```diff
@@ -58,6 +72,7 @@ export function createProgram(options: ProgramOptions): Program {
 			outputs: outputs,
 			diagnostics: diagnostics,
 			sourceFiles: [],
+			configIdsMap: {},
```

Comment:
It would be better to specify exactly that the type is dictionary of key: string, any/string(value).
{ [key: string]: any } or { [key: string]: string }

Reply:
Done (in types.ts)